docs/content/misc/version-support-status.ngdoc

```diff
@@ -7,7 +7,7 @@
 This page describes the support status of the significant versions of AngularJS.
 
 <div class="alert alert-info">
-  On July 1, 2018 AngularJS entered a 3 year Long Term Support period.
+  On July 1, 2018 AngularJS entered a 3 year Long Term Support period. Due to COVID-19 causing delays to a number of teams going through the conversion, the support will be extended for an additional 6 months.
```

Comment:
I think it would be better to make it clear that this is an update to the original info. For example:

```suggestion
  On July 1, 2018 AngularJS entered a 3 year Long Term Support period.<br />
  <br />
  _**UPDATE (2020-07-27):**_<br />
  _Due to COVID-19 affecting teams migrating from AngularJS, we are extending the LTS by 6 months
  (until the end of 2021)._
```

...which will be rendered as:

![lts-entension](https://user-images.githubusercontent.com/8604205/88645124-726eef00-d0cc-11ea-9305-573f8dc04cf4.png)